components/atoms/TextInput/text-input.tsx

```diff
@@ -62,7 +62,7 @@ const TextInput = ({
             id={id || name || ""}
             placeholder={placeholder || ""}
             className={`flex-1 focus:outline-none  ${
```

Comment:
LGTM :pizza: 

[nit]: use clsx to make it more readable
```suggestion
               className={clsx(
              "flex-1 focus:outline-none",
              disabled && "bg-light-slate-3 cursor-not-allowed text-light-slate-9",
              classNames
            )}
```

Reply:
@takanome-dev I have made the changes